tests/models/cassettes/test_model_names/test_known_model_names.yaml

```diff
@@ -99,53 +99,19 @@ interactions:
       alt-svc:
       - h3=":443"; ma=86400
       content-length:
-      - '762'
+      - '99'
       content-type:
       - application/json
       referrer-policy:
       - strict-origin-when-cross-origin
       strict-transport-security:
       - max-age=3600; includeSubDomains
     parsed_body:
-      data:
-      - created: 0
-        id: llama3.1-8b
-        object: model
-        owned_by: Cerebras
-      - created: 0
-        id: qwen-3-coder-480b
-        object: model
-        owned_by: Cerebras
-      - created: 0
-        id: llama-4-maverick-17b-128e-instruct
-        object: model
-        owned_by: Cerebras
-      - created: 0
-        id: qwen-3-235b-a22b-thinking-2507
-        object: model
-        owned_by: Cerebras
-      - created: 0
-        id: llama-3.3-70b
-        object: model
-        owned_by: Cerebras
-      - created: 0
-        id: qwen-3-32b
-        object: model
-        owned_by: Cerebras
-      - created: 0
-        id: qwen-3-235b-a22b-instruct-2507
-        object: model
-        owned_by: Cerebras
-      - created: 0
-        id: llama-4-scout-17b-16e-instruct
-        object: model
-        owned_by: Cerebras
-      - created: 0
-        id: gpt-oss-120b
-        object: model
-        owned_by: Cerebras
-      object: list
+      code: wrong_api_key
```

Comment:
Please remove the changes in this file

Reply:
Note: This file changes every time I run the make test locally